### ISSUE TITLE
[Feature] Development ready testing environment

### ISSUE THREAD

**Issue by quank123wip** [Milkdown/milkdown#1434]
### Initial checklist

- [X] I agree to follow the [code of conduct](https://github.com/Saul-Mirone/milkdown/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched [issues](hhttps://github.com/Saul-Mirone/milkdown/issues) and [discussions](https://github.com/Saul-Mirone/milkdown/discussions) and couldn’t find anything (or linked relevant results below)

### Problem

As milkdown has rich support for plugins and themes, maybe provide a official easy-to-config, easy-to-fetch, easy-to-moddify environment contains necessary packages and configs for developers testing their plugins and themes.

### Solution

Just start from milkdown/e2e, create a template project for developers to begin on.

### Alternatives

For now, people can make their own development environment with vite or something else on the basis of e2e.

**Comment by Saul-Mirone** [comment]
I'm working on [crepe](https://milkdown-storybook.vercel.app/?path=/story/crepe-crepe--with-default-value). Is this something what you want?

**Comment by quank123wip** [comment]
> I'm working on [crepe](https://milkdown-storybook.vercel.app/?path=/story/crepe-crepe--with-default-value). Is this something what you want?

Not quite, crepe makes this easier, but imo we need a template repository to let developers start faster. Maybe we just need to publish a repository shipped with crepe and other configs.
